docs/_layouts/default.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE HTML>
+<html lang="" >
+    <head>
+        <title>{{ page.title }}</title>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+        <meta name="description" content="">
```

Comment:
Not sure it is useful as always empty. We should either remove it, or fill a new `description` field on each documentation page for SEO.